fastlane/AppFile

```diff
@@ -0,0 +1,4 @@
+app_identifier "io.cozy.files"
```

Comment:
I'd prefer `io.cozy.mobile.sync`, as it will probably sync not only files in a few months.